traveler-ios-ui/TravelerKitUI/ModelExtensions/Product.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Product.swift
```

Comment:
Id jus add something here to denote whats in the file i.e `Product+Description.swift`